docs/ecosystem/nats/index.mdx

```diff
@@ -0,0 +1,88 @@
+---
+title: "NATS"
+date: 2025-05-29T00:00:00+00:00
+icon: "ti-map" # themify icon pack : https://themify.me/themify-icons
+description: "NATS connective technology for distributed systems"
+type: "docs"
+sidebar_position: 3
+---
+
+# NATS
+
+## Overview
+
+The wasmCloud [lattice](/docs/concepts/lattice) is built on [**NATS**](https://nats.io/), an open source mesh networking technology hosted by the Cloud Native Computing Foundation (CNCF). NATS enables secure application-layer networking across diverse environments including edge, different vendors' clouds, and on-premise datacenters. 
```

Comment:
I think we should avoid calling this a "mesh network" as it isn't really that. I think we should either echo their language (Connective Technology for Adaptive Edge & Distributed Systems) which I don't really like or we should call it a "messaging and streaming system" (wording is just an idea, I'm not married to it